packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -618,10 +618,10 @@ $screen-lg: 992px;
 $screen-xl: 1200px;
 
 // So media queries don't overlap when required, provide a maximum (used for max-width)
-$screen-xs-max: ($screen-sm - 1);
-$screen-sm-max: ($screen-md - 1);
-$screen-md-max: ($screen-lg - 1);
-$screen-lg-max: ($screen-xl - 1);
+$screen-xs-max: calc($screen-sm - 1);
```

Comment:
Maybe my comment is not related to PR itself, cuz the code was there before, but I think we have an issue here, and this whole group of variables are invalid.
Because when you use calc with `+` or `-` operators the units should be specified.
[MDN calc()](https://developer.mozilla.org/en-US/docs/Web/CSS/calc())

Reply:
TIL, will fix it

Reply:
Fixed with force-push